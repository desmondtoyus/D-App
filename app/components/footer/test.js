import React from 'react';
import {render, cleanup} from '../../../__tests__/setup/test-setup';
import Footer from './index';

afterEach(cleanup);

describe('<Footer', () => {
    it('should match snapshot', () => {
        const {asFragment} = render(<Footer/>);
        expect(asFragment).toMatchSnapshot();
    });
    it('should contain footer class', () => {
        const {getByTestId} = render(<Footer/>);
        const footer = getByTestId("nb-footer");
        expect(footer.classList.contains("nb-footer")).toBe(true);

    });
    it('should have active links', () => {
        const {getByTestId} = render(<Footer/>);
        expect(getByTestId('linkedin').getAttribute('href')).toEqual('https://www.linkedin.com/in/desmondademiluyi/');
        expect(getByTestId('facebook').getAttribute('href')).toEqual('https://www.facebook.com/ademiluyi.toye');
        expect(getByTestId('github').getAttribute('href')).toEqual('https://github.com/desmondtoyus');
        expect(getByTestId('twitter').getAttribute('href')).toEqual('https://twitter.com/arewethere_yt');
    });
});