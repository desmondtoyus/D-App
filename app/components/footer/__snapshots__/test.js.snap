// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer should load successfully 1`] = `[Function]`;

exports[`<Footer should match snapshot 1`] = `[Function]`;
